envoy-control-core/src/test/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/SnapshotUpdaterTest.kt

```diff
@@ -83,8 +83,6 @@ class SnapshotUpdaterTest {
 
         hasSnapshot(cache, groupWithProxy)
             .hasClusters("existingService1", "existingService2")
-            .hasSecuredIngressRoute("/endpoint", "client")
-            .hasServiceNameRequestHeader("service")
```

Comment:
If we remove these assertions, we should either:
1. Remove unnecessary parts of proxySettings defined at the beginning.
2. Add corresponding assertions for RBAC

Reply:
(1) is still not fixed as I see.